pkg/lib/qunit-tests.ts

```diff
@@ -28,19 +28,11 @@ export const mock_info = async (key: string) => {
     return (await response.json())[key];
 };
 
-// Convenience for skipping tests that the python bridge can't yet
-// handle.
-
-let is_pybridge: boolean | null = null;
+// Convenience for skipping tests that our python bridge can't yet
+// handle (the C bridge implemented these features)
 
 export const skipWithPybridge = async (name: string, callback: (assert: unknown) => void | Promise<void>) => {
```

Comment:
But did we ever want to implement those features, we used to have todoPybridge in our integration tests

Reply:
I don't know -- but I don't feel like just dropping all these tests. Obviously they are not very important, but we should at least go through them with a fine comb at some point.

Reply:
Agreed, this feels like a useful thing to do